src/android/nl/xservices/plugins/accessor/AbstractCalendarAccessor.java

```diff
@@ -591,7 +591,7 @@ public String createCalendar(String calendarName, String calendarColor) {
             Cursor result = contentResolver.query(evuri, new String[]{CalendarContract.Calendars._ID, CalendarContract.Calendars.NAME, CalendarContract.Calendars.CALENDAR_DISPLAY_NAME}, null, null, null);
             if (result != null) {
                 while (result.moveToNext()) {
-                    if (result.getString(1).equals(calendarName) || result.getString(2).equals(calendarName)) {
+                    if ((result.getString(1) != null && result.getString(1).equals(calendarName)) || (result.getString(2) != null && result.getString(2).equals(calendarName))) {
```

Comment:
We could invert the comparison to avoid a null check: `calendarName.equals(result.getString(1))`, but this does the job as well.